MODULE.bazel

```diff
@@ -1,4 +1,4 @@
 module(name = "protobuf_javascript", version = "3.21.2")
 
-bazel_dep(name = "protobuf", version = "21.7", repo_name = "com_google_protobuf")
+bazel_dep(name = "protobuf", version = "23.1", repo_name = "com_google_protobuf")
```

Comment:
Note that 23.1 is the latest available through bzlmod, but this is tested to also work with 25.x on our internal system.

Reply:
You can use a specific version of the module if it’s not yet published to BCR with [`git_override()`](https://bazel.build/rules/lib/globals/module#git_override), like this:

```bazel
bazel_dep(name = "protobuf", version = "25.3", repo_name = "com_google_protobuf")

git_override(
    module_name = "protobuf",
    remote = "https://github.com/protocolbuffers/protobuf.git",
    # https://github.com/protocolbuffers/protobuf/releases/tag/v25.3
    commit = "4a2aef570deb2bfb8927426558701e8bfc26f2a4",
)
```

or with [`archive_override()`](https://bazel.build/rules/lib/globals/module#archive_override), like this:

```bazel
bazel_dep(name = "protobuf", version = "25.3", repo_name = "com_google_protobuf")

archive_override(
    module_name = "protobuf",
    urls = ["https://github.com/protocolbuffers/protobuf/releases/download/v25.3/protobuf-25.3.tar.gz"],
    integrity = "sha256-0ZZD0mW5eDgzUrMUPwTAZB7qdadSNcERzAGhNQFzGA4=",
    strip_prefix = "protobuf-25.3",
)
```